app/actions/device/index.js

```diff
@@ -2,16 +2,20 @@
 // See LICENSE.txt for license information.
 
 import {networkStatusChangedAction} from 'redux-offline';
+import {batchActions} from 'redux-batched-actions';
 
 import {DeviceTypes} from 'app/constants';
 
 export function connection(isOnline) {
     return async (dispatch) => {
-        dispatch(networkStatusChangedAction(isOnline));
-        dispatch({
-            type: DeviceTypes.CONNECTION_CHANGED,
-            data: isOnline,
-        });
+        if (isOnline !== undefined) { //eslint-disable-line no-undefined
+            dispatch(batchActions([
+                networkStatusChangedAction(isOnline), {
+                    type: DeviceTypes.CONNECTION_CHANGED,
+                    data: isOnline,
```

Comment:
Recommend adding a name to batched actions.
(Leaving as single review comment for all `batchAction` calls in PR.)

Reply:
DONE!!!! please take a look

Reply:
Awesome. And the `BATCH_` prefix makes sense too. Also, easier to find in code searches. Thanks.